docs/api/server/Environment-Variables.md

```diff
@@ -29,13 +29,15 @@ One App can be configured via Environment Variables:
   * [`ONE_CLIENT_ROOT_MODULE_NAME`](#one_client_root_module_name) ⚠️
   * [`ONE_CLIENT_CDN_URL`](#one_client_cdn_url) ⚠️
   * [`ONE_CONFIG_ENV`](#one_config_env) ⚠️
+  * [`ONE_CONFIG_USE_NATIVE_INTL`](#one_config_use_native_intl)
 * Running in Development
   * [`NODE_ENV`](#node_env) ⚠️
   * [`ONE_CLIENT_ROOT_MODULE_NAME`](#one_client_root_module_name) ⚠️
   * [`ONE_CONFIG_ENV`](#one_config_env) ⚠️
   * [`ONE_DANGEROUSLY_ACCEPT_BREAKING_EXTERNALS`](#ONE_DANGEROUSLY_ACCEPT_BREAKING_EXTERNALS)
   * [`ONE_CSP_ALLOW_INLINE_SCRIPTS`](#ONE_CSP_ALLOW_INLINE_SCRIPTS)
   * [`ONE_DANGEROUSLY_DISABLE_CSP`](#ONE_DANGEROUSLY_DISABLE_CSP)
+  * * [`ONE_CONFIG_USE_NATIVE_INTL`](#one_config_use_native_intl)
```

Comment:
```suggestion
  * [`ONE_CONFIG_USE_NATIVE_INTL`](#one_config_use_native_intl)
```

Reply:
Sure, fixed in [65a6992](https://github.com/americanexpress/one-app/pull/1258/commits/65a699202bdf803beb95baac0a58f2a5c4e24ae7)